src/utils/messages.ts

```diff
@@ -181,3 +181,17 @@ export function getSenderUserIdFromMessage(
   // @ts-ignore
   return message?.sender?.userId ?? undefined;
 }
+
+const msgFilter = {
+  sys: {
```

Comment:
Q. Do we have reason for having `sys` here?

Reply:
I wanted to express that I'm filtering messages sent as system messages.
Using "systemMessage" as the key increases the module size, so I wrote it as "sys" instead ㅜ0ㅜ

Reply:
🤦‍♀️